src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -157,25 +160,27 @@ public ClimbSubsystem() {
         */
 
         // Initialize Shuffleboard entries
-        shuffleboardTab = Shuffleboard.getTab("Climb");
+        shuffleboardTab = new GRTShuffleboardTab("Climb");
+        sixPosEntry = shuffleboardTab.addEntry("Six pos", sixEncoder.getPosition());
 
         // If DEBUG_PID is set, allow for PID tuning on shuffleboard
         if (DEBUG_PID) {
-            shuffleboardTab.add("Six kP", sixP).getEntry()
-                .addListener(this::setSixP, EntryListenerFlags.kUpdate);
-            shuffleboardTab.add("Six kI", sixI).getEntry()
-                .addListener(this::setSixI, EntryListenerFlags.kUpdate);
-            shuffleboardTab.add("Six kD", sixD).getEntry()
-                .addListener(this::setSixD, EntryListenerFlags.kUpdate);
-            shuffleboardTab.add("Six kFF", sixFF).getEntry()
-                .addListener(this::setSixFF, EntryListenerFlags.kUpdate);
-            shuffleboardTab.add("Six maxVel", maxVel).getEntry()
-                .addListener(this::setSixMaxVel, EntryListenerFlags.kUpdate);
-            shuffleboardTab.add("Six maxAccel", maxAccel).getEntry()
-                .addListener(this::setSixMaxAccel, EntryListenerFlags.kUpdate);
+            shuffleboardTab
+                .list("Six PID")
+                .addListener("kP", sixP, this::setSixP)
+                .addListener("kI", sixI, this::setSixI)
+                .addListener("kD", sixD, this::setSixD)
+                .addListener("kFF", sixFF, this::setSixFF)
+                .addListener("Max vel", maxVel, this::setSixMaxVel)
+                .addListener("Max accel", maxAccel, this::setSixMaxAccel);
         }
     }
 
+    @Override
+    public void periodic() {
+        sixPosEntry.setValue(sixEncoder.getPosition());
```

Comment:
i feel like setting motor powers should go in periodic. this means we have a state with the desired power, and in periodic we decide what to do with it